<launch>
	<arg name="model" />
	<arg name="gui" default="False" />
	<param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
	<param name="use_gui" value="$(arg gui)" />
	<node pkg="testbot_description" name="control_robot" type="control_robot.py"/>
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find testbot_description)/urdf.rviz" required="true"/>
</launch>
